### ISSUE TITLE
Nested Array should be supported

### ISSUE THREAD

**Issue by GoogleCodeExporter** [runt18/google-bigquery#150]
```
I have faced the issue with nested array in a JSON which has not been accepted 
by BigQuery.For example- the below JSON (part of original) shows 'coordinates' 
field in multi-level array.I could not resolve it in BigQuery schema.

  "location": {
    "link": "some-text",
    "geo": {
      "type": "Polygon",
      "coordinates": [
        [
          [
            -122.82613,
            45.393108
          ],
          [
            -122.82613,
            45.456634
          ],
          [
            -122.74354,
            45.456634
          ],
          [
            -122.74354,
            45.393108
          ]
        ]
      ]
    }
  }
There should be a work-around for this. This type of JSON is very normal for 
any kind of social media output for analytics.
```

Original issue reported on code.google.com by `subhayu....@abzooba.com` on 9 Sep 2014 at 6:25

**Comment by GoogleCodeExporter** [comment]
```
Thank you for the request.  We may support this type of structure in the future.

In the meantime, you could create repeated column of type RECORD to store this 
type of data:

coordinates : [
          {
            lat: -122.82613,
            lon: 45.393108
          },
          {
            lat: -122.82613,
            lon: 45.456634
          },
          {
            lat: -122.74354,
            lon: 45.456634
          },
          {
            lat: -122.74354,
            lon: 45.393108
          }
]
```

Original comment by `thomasp...@google.com` on 25 Nov 2015 at 1:07
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
```
hi,  any idea if this is fixed? thanks
```

Original comment by `goelt2...@gmail.com` on 24 Feb 2016 at 8:08

**Comment by GoogleCodeExporter** [comment]
```
> hi,  any idea if this is fixed? thanks

The suggested fix at this time is in #1 - create an array of lat/long objects.

We're working on SQL dialect improvements 
(https://code.google.com/p/google-bigquery/issues/detail?id=448) that will also 
encompass more flexible schema handling for UDFs, so please follow that issue 
if you'd like to get updates.
```

Original comment by `thomasp...@google.com` on 21 Mar 2016 at 6:47
- Changed state: **WontFix**
